src/main/java/Contacts0.java

```diff
@@ -146,4 +146,12 @@ public static void main(String[] args) {
         }
     }
 
+    private static void showWelcomeMessage() {
```

Comment:
Very nice addition. Keep it up!

Reply:
Thanks Andy! Inspired by your changes actually!